components/scream/src/physics/shoc/atmosphere_macrophysics.cpp

```diff
@@ -164,49 +167,52 @@ void SHOCMacrophysics::initialize_impl (const util::TimeStamp& t0)
           exner("exner",m_num_cols,nlev_packs),
           thlm("thlm",m_num_cols,nlev_packs),
           qw("qw",m_num_cols,nlev_packs),
-          cloud_frac("cloud_frac",m_num_cols,nlev_packs);
+          qv_copy("qv_copy",m_num_cols,nlev_packs),
+          qc_copy("qc_copy",m_num_cols,nlev_packs),
```

Comment:
Can you add a TODO comment (here, or maybe in the hpp file), reminding us to remove the `*_copy` vars once shoc can request a subgroup of the tracers?

Reply:
I'll make a small PR.